piccolo/columns/constraints.py

```diff
@@ -0,0 +1,42 @@
+from .base import ColumnMeta, Column
+
+class Constraint(Column):
+    def __init__(self) -> None:
+        pass
+
+class UniqueConstraint(Constraint):
+    """
+    This class represents UNIQUE CONSTRAINT, which can be use to create
+    many complex (multi-field) constraints for the Table
+    All manipulations with the Constraint are like anything about Columns
+    
+    Usage:
+        class FooTable(Table):
+            foo_field = Text()
+            bar_field = Text()
+            my_constraint_1 = UniqueConstraint(['foo_field','bar_field'])
+
+    SQL queries for creating and dropping constrains are similar to:
+        ALTER TABLE foo_table ADD CONSTRAINT my_constraint_1 UNIQUE (foo_field, bar_field);
+        ALTER TABLE foo_table DROP IF EXIST CONSTRAINT my_constraint_1;
+    """
+    def __init__(self, unique_columns: list[str]) -> None:
```

Comment:
I think this is why the tests are failing.

We can't use this newer syntax yet, because we're keeping backwards compatibility with Python 3.7.

So it'll have to be `Typing[str]` instead.

Reply:
Sorry, I used to Python3.10, so I forgot about older versions